src/Mvc/Mvc.Core/src/PublicAPI.Unshipped.txt

```diff
@@ -544,6 +545,8 @@ Microsoft.AspNetCore.Mvc.Infrastructure.ActionDescriptorCollection.Items.get ->
 Microsoft.AspNetCore.Mvc.Infrastructure.AmbiguousActionException.AmbiguousActionException(System.Runtime.Serialization.SerializationInfo! info, System.Runtime.Serialization.StreamingContext context) -> void
 Microsoft.AspNetCore.Mvc.Infrastructure.AmbiguousActionException.AmbiguousActionException(string? message) -> void
 Microsoft.AspNetCore.Mvc.Infrastructure.ContentResultExecutor.ContentResultExecutor(Microsoft.Extensions.Logging.ILogger<Microsoft.AspNetCore.Mvc.Infrastructure.ContentResultExecutor!>! logger, Microsoft.AspNetCore.Mvc.Infrastructure.IHttpResponseStreamWriterFactory! httpResponseStreamWriterFactory) -> void
+static Microsoft.AspNetCore.Mvc.Core.Routing.UrlHelpers.Content(Microsoft.AspNetCore.Http.HttpContext! httpContext, string? contentPath) -> string?
+static Microsoft.AspNetCore.Mvc.Core.Routing.UrlHelpers.IsLocalUrl(string? url) -> bool
```

Comment:
Let's try to keep the public API changes to a minimum. If you think we should add these, I suggest submitting an [API proposal](https://github.com/dotnet/aspnetcore/issues/new/choose).

Reply:
Sure, based on the answer to the previos comment I might not need any of these. thanks!